Makefile

```diff
@@ -1,8 +1,10 @@
+UNITS = "@unit.abijson or @unit.algod or @unit.applications or @unit.atomic_transaction_composer or @unit.dryrun or @unit.feetest or @unit.indexer or @unit.indexer.logs or @unit.offline or @unit.rekey or @unit.responses or @unit.responses.231 or @unit.tealsign or @unit.transactions or @unit.transactions.payment"
```

Comment:
Introduce variables for the cucumber test tags, and also alphabetize. 

This allows usage such as:

```bash
make unit UNITS="@unit.atomic_transaction_composer"
```